polkadot/xcm/pallet-xcm/src/mock.rs

```diff
@@ -172,7 +172,7 @@ impl SendXcm for TestSendXcm {
 		msg: &mut Option<Xcm<()>>,
 	) -> SendResult<(Location, Xcm<()>)> {
 		if FAIL_SEND_XCM.with(|q| *q.borrow()) {
-			return Err(SendError::Transport("Intentional send failure used in tests"))
+			return Err(SendError::Transport("Intentional send failure used in tests"));
```

Comment:
I think you are using a different rustfmt version?

Reply:
I think I am. Interestingly enough, the rustfmt check doesn't break with just these semicolons.